grammars/javascript.cson

```diff
@@ -702,11 +702,11 @@
   {
     # console
     # console.log(arg1, "arg2", [...])
-    'begin': '\\bconsole\\b'
+    'begin': '\\bconsole\\b(?!\\$)'
     'beginCaptures':
       '0':
         'name': 'entity.name.type.object.console.js'
-    'end': '(?!\\G)'
+    'end': '(?=(?!\\s*\\.)\\S)'
     'patterns': [
       {
         'begin': '\\s*(\\.)\\s*(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)\\s*(?=\\()'
```

Comment:
Here we can capture all properties?

`'begin': '\\s*(\\.)\\s*(\w+)\\s*(?=\\()'`